pages/about.html

```diff
@@ -46,11 +45,11 @@
           
         <div class="desktop-menu">
             <ul class="menu-items">
-            <li><a href="./pages/about.html"  class="mitem">About</a></li>
-            <li><a href="#features"  class="mitem">Features</a></li>
-            <li><a href="#partners"  class="mitem">Partners</a></li>
-            <li><a href="./pages/about.html#email"  class="mitem">Contact Us</a></li>
-            <li><a href="#speaker-container"  class="mitem">Clients</a></li>
+            <li><a href="../pages/about.html"  class="mitem">About</a></li>
```

Comment:
- [ ] In the main program in the desktop version I can see horizontal scrolling but I think it would better if you remove it since it dpesn't exist in the original design you can use overflow-x:hidden for that 🔥🔥🙏🏻

- [ ] Kindly make the width and the height for the cards container to be bigger and the cards itself decrease the width and the height to make it very simialr to the original one kindly work on that to be fixed well

- your design
![maf](https://user-images.githubusercontent.com/58553711/181492054-a6880cf8-bb56-4acb-a037-4019b29a43bf.PNG)



- Original design
![image](https://user-images.githubusercontent.com/58553711/181491244-5b05277d-5691-4622-b8ae-c0ab4aa1588a.png)